Fluid.MvcViewEngine/FluidRendering.cs

```diff
@@ -62,6 +62,9 @@ static FluidRendering()
             context.AmbientValues[ViewPath] = path;
             context.AmbientValues.Add("Sections", new Dictionary<string, IList<Statement>>());
 
+            var viewsPath = Path.Combine(_hostingEnvironment.ContentRootPath, "Views");
+            context.FileProvider = new PhysicalFileProvider(viewsPath);
```

Comment:
I used `PhysicalFileProvider` which I dislike because the `fileProvider` will point to the root and I'm not able to get the views path from the `IncludeStatment`

IMHO we need to add a property in the `Options` to let the devs specify the partials folder, it's similar to what you said before